diff --git a/dist/index.cjs b/dist/index.cjs
index c644d020776c4de1dc129638629af227c28cf81a..fd0b5b386cd5dd9ac5bc3b11eaf40fa1c12c2b50 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -70,6 +70,7 @@ function package_default(options = {}) {
         return null;
       }
       if (shouldProcess(qs)) {
+        this.addWatchFile(path);
         let code = await (0, import_promises.readFile)(path, { encoding: "utf8" });
         if (svgo.enabled) {
           let optimized = await optimizeSvg(code, path, svgo.svgoConfig);
diff --git a/dist/index.js b/dist/index.js
index 913e0fad1f52e8a96b9ec0f635b3c9b7a11d9db4..f83aab195ccd807b3cdb30e190b9fd2badb65687 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -47,6 +47,7 @@ function package_default(options = {}) {
         return null;
       }
       if (shouldProcess(qs)) {
+        this.addWatchFile(path);
         let code = await readFile(path, { encoding: "utf8" });
         if (svgo.enabled) {
           let optimized = await optimizeSvg(code, path, svgo.svgoConfig);
